presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7057,30 +7059,52 @@ public void testExistsSubqueryWithGroupBy()
     @Test
     public void testCorrelatedScalarSubqueries()
     {
-        String errorMsg = "Unexpected node: com.facebook.presto.sql.planner.plan.LateralJoinNode";
-
-        assertQueryFails("SELECT (SELECT l.orderkey) FROM lineitem l", errorMsg);
-        assertQueryFails("SELECT (SELECT 2 * l.orderkey) FROM lineitem l", errorMsg);
-        assertQueryFails("SELECT * FROM lineitem l WHERE 1 = (SELECT 2 * l.orderkey)", errorMsg);
-        assertQueryFails("SELECT * FROM lineitem l ORDER BY (SELECT 2 * l.orderkey)", errorMsg);
+        assertQueryFails(
+                "SELECT (SELECT l.orderkey) FROM lineitem l",
+                UNSUPPORTED_CORRELATED_SUBQUERY_ERROR_MSG);
```

Comment:
You should be testing (at least in a few cases) that the error is reported at the correct location.